laravel/app/Http/Controllers/SlotController.php

```diff
@@ -55,6 +55,11 @@ private function userAllowed(Slot $slot)
             }
         }
 
+        if($slotRoles->isEmpty())
```

Comment:
this allows banned users to sign up for shifts. i would rework this logic slightly and check for banned